src/meet-calendar.ts

```diff
@@ -0,0 +1,688 @@
+    // to create event page
+    if (
+      document.querySelector("body")?.dataset.viewfamily === "EVENT_EDIT" &&
+      !c.isButtonPresent()
+    ) {
+      // popup adds autoCreateMeeting param when open directly event
+      // create page
+      if (
+        findGetParameter("autoCreateMeeting") &&
+        findGetParameter("extid") === chrome.runtime.id
+      ) {
+        c.scheduleAutoCreateMeeting = true;
+      }
+
+      c.update();
+    }
+
+    // Listen for mutations (showing the bubble), for quick adding events
+    var body = document.querySelector("body")!;
+    new MutationObserver(function (mutations) {
+      // the main calendar view
+      if (body.dataset.viewfamily === "EVENT") {
+        mutations.forEach(function (mutation) {
+          var mel = mutation.addedNodes[0];
+          var newElement = mel && (mel as HTMLElement).innerHTML;
+
+          if (newElement && newElement.search('role="dialog"') !== -1) {
+            // skip if our button is already added
+            if ($("#jitsi_button_quick_add").length != 0) {
+              return;
+            }
+
+            var tabEvent = $(mel).find("#tabEvent");
+            if (tabEvent.length > 0) {
+              const container = $(
+                '<content class="" role="tabpanel" id="jitsi_button_quick_add_content"><div class="fy8IH poWrGb"/></content>'
+              );
+              $(tabEvent.parent()).append(container);
+
+              container.children().first().append(`
+                                <div class="FkXdCf HyA7Fb">
+                                    <div class="DPvwYc QusFJf jitsi_quick_add_icon"/>
+                                </div>
+                            `);
+
+              container.children().first().append(`
+                                <div class="mH89We">
+                                    <div role="button"
+                                        class="uArJ5e UQuaGc Y5sE8d"
+                                        id="jitsi_button_quick_add">
+                                        <content class="CwaK9">\
+                                            <span class="RveJvd jitsi_quick_add_text_size">
+                                                Add a ${LOCATION_TEXT}
+                                            </span>
+                                        </content>
+                                    </div>
+                                </div>
+                            `);
+
+              var clickHandler = container.find("#jitsi_button_quick_add");
+              clickHandler.on("click", function (e) {
+                c!.scheduleAutoCreateMeeting = true;
+                $('div[role="button"][jsname="rhPddf"]').trigger("click");
+              });
+
+              return;
+            }
+          }
+        });
+      } else if (
+        document.querySelector("body")?.dataset.viewfamily === "EVENT_EDIT"
+      ) {
+        c!.update();
+      }
+    }).observe(body, {
```

Comment:
we have had some issues in the past with MutationObservers causing page performance issues. you may want to measure page load time before and after this change to make sure it's not a big difference.

Reply:
Good suggestion, I will do this.